README.md

```diff
@@ -149,11 +149,22 @@ Instances of `Record` and `Tuple` are immutable, so their equality works like th
 
 ```js
 assert(#{ a: 1 } === #{ a: 1 });
-assert(Object(#{ a: 1 }) !== Object(#{ a: 1 }));
+assert(#[1, 2] === #[1, 2]);
+```
+
+This is distinct from how equality works for JS objects. Strict comparison of objects will observe that each object is distinct: 
+
+```js
 assert({ a: 1 } !== { a: 1 });
+assert(Object(#{ a: 1 }) !== Object(#{ a: 1 }));
+assert(Object(#[1, 2]) !== Object(#[1, 2]));
 ```
 
-This is distinct from how equality works for JS objects: `assert({} !== {});`. Strict comparison of objects will observe that each object is distinct.
+Insertion order does not affect equality of records:
```

Comment:
this implies that it does affect equality of tuples; can an example be provided?

Reply:
rewrote to "Insertion order of record keys does not affect equality of records"